examples/group_vars/all.yml

```diff
@@ -32,10 +32,19 @@ gpu_dp_namespace: kube-system
 # Forces installation of the Multus CNI from the official Github repo on top of the Kubespray built-in one
 force_external_multus_installation: true
 
+# Create reference net-attach-def objects
+example_net_attach_defs:
```

Comment:
Consider changing the default to false, as I imagine it can cause issues if the userspace parts are not installed

Reply:
actually net-attach-def won't do anything as long as the user doesn't reference it in their pod spec, but yea, it makes sense to not create them by default